open-apparel-registry/open-apparel-registry
# Update facility static map marker

## Overview

This PR replaces the legacy static selected marker with one that is styled to match the rest of OS Hub.

Connects #2132 

## Demo

![Screen Shot 2022-09-26 at 2 58 23 PM](https://user-images.githubusercontent.com/8356789/192358004-4d471505-cbe3-48d3-a50f-be9ee1aff019.png)

## Notes

This can't be tested locally because google will make a request to fetch the marker icon which fails because it cannot access localhost where the image is hosted.

## Testing Instructions

* No way to test locally.
* In staging, go to a facility details page.
* Check that static map marker is updated

## Checklist
